node/src/BaseClient.ts

```diff
@@ -3630,6 +3632,80 @@ export class BaseClient {
         return this.createWritePromise(createXLen(key));
     }
 
+    /**
+     * Returns information about the stream stored at `key`.
+     *
+     * @param key - The key of the stream.
+     * @param fullOptions - If `true`, returns verbose information with a limit of the first 10 PEL entries.
+     * If `number` is specified, returns verbose information limiting the returned PEL entries.
+     * If `0` is specified, returns verbose information with no limit.
+     * @returns A map of detailed stream information for the given `key`. See
```

Comment:
```suggestion
     * @returns A `Record` of detailed stream information for the given `key`. See
```

Reply:
The different between a Record and a map is subtle, and this is a lower-case 'm' map. I think this is why it's being used in other stream commands. I don't feel strongly about either and can change it to 'Record'.